store/sqlstore/upgrade.go

```diff
@@ -350,6 +351,17 @@ func UpgradeDatabaseToVersion47(sqlStore SqlStore) {
 		sqlStore.AlterColumnTypeIfExists("OAuthAuthData", "State", "varchar(1024)", "varchar(1024)")
 		sqlStore.RemoveColumnIfExists("ChannelMemberHistory", "Email")
 		sqlStore.RemoveColumnIfExists("ChannelMemberHistory", "Username")
+
+		defaultTimezone := make(map[string]string)
```

Comment:
I think it might make sense to move the code that makes this default timezone map into `model/user.go` in case we need to construct an object like that elsewhere.

Reply:
Yeh, this would clean it up to be used elsewhere if needed.